doc/plugins/proxy-rewrite.md

```diff
@@ -35,6 +35,7 @@ upstream proxy info rewrite plugin.
 |-------         |-----|------|
 |scheme          |optional| Upstream new `schema` forwarding protocol,options can be `http` or `https`,default `http`.|
 |uri             |optional| Upstream new `uri` forwarding address.|
+|regex_uri       |optional| Upstream new `uri` forwarding address. Use regular expression to match uri from client, when the match is successful, the uri template will be forwarded upstream. If the match is not successful, the uri from the client will be forwarded to the upstream. When `uri` and` regex_uri` are both exist, `uri` is used first. For example: [" ^/iresty/(.*)/(.*)/(.*)", "/$1-$2-$3"], the first element represents the matching regular expression and the second element represents the uri template that is forwarded to the upstream.|
```

Comment:
> If the match is not successful, the uri from the client will be forwarded to the upstream.

If the match fails, is it expected to continue using the original request uri? What do you think of this?

Reply:
Similar to this situation in Nginx.
```nginx
server {
	listen 80;
	location / {
	    rewrite ^/hello/(.*) /$1 break;
	    proxy_pass http://127.0.0.1:8080;
	}
}

server {
	listen 8080;
	location / {
		return 200, $uri;
	}
}

$curl http://127.0.0.1/hello/world
Output: /world

$curl http://127.0.0.1/other/things
Output: /other/things
```

Reply:
thank you for the explanation.